pkg/satellite/stream/stream.go

```diff
@@ -271,6 +295,14 @@ func (ss *satelliteStream) recvLoop() {
 			metrics.setStreamId(ss.streamId)
 		}
 
+		latestByteTime, _ := ptypes.Timestamp(timestampLastByteReceived)
+		if ss.enableAutoClose && ss.autoCloseTime.Sub(latestByteTime) < 1*time.Second && !receivingBytes {
+			autoCloseTimer.Stop()
```

Comment:
If there are long stretches of no data, should we be concerned if the timer is reset in a tight loop like this?